specifyweb/frontend/js_src/lib/components/QueryBuilder/__tests__/fromTree.test.ts

```diff
@@ -44,6 +49,7 @@ allTrees.map((table, rankId) => {
 test('queryFromTree', async () =>
   expect(
     Promise.all(
-      allTrees.map(async (tree, index) => queryFromTree(tree, index * 4))
+      // TODO: Add static data for Tectonic Unit and replace testingTrees with allTrees
```

Comment:
reminder

Reply:
Looks like this specific test is for the `queryFromTree` function, which only has placeholder fields for now for Tectonic trees. Might have to update this test later when we update `queryFromTree`